crates/volta-core/src/tool/package/mod.rs

```diff
@@ -163,6 +169,11 @@ pub fn uninstall(name: &str) -> Fallible<()> {
             .with_context(delete_dir_error(&package_image_dir))?;
     }
 
+    match package_found {
+        true => info!("{} package '{}' uninstalled", success_prefix(), name),
+        false => warn!("No package '{}' found to uninstall", name),
+    };
```

Comment:
For `true/false`, matching looks a bit odd imo (I also think there might be a clippy lint warning around it). Instead we should try to use `if package_found {} else {}`, as it has the same effect and is slightly more readable.